ExchangeSharp/API/Exchanges/ExchangeBinanceAPI.cs

```diff
@@ -367,16 +372,39 @@ public override IEnumerable<MarketCandle> GetCandles(string symbol, int periodSe
 
         public override ExchangeOrderResult PlaceOrder(ExchangeOrderRequest order)
```

Comment:
Would it be worth also catching the specific errors thrown by the Binance webservice (LOT_SIZE/PRICE_FILTER) and reissuing the request?